site/pages/technology/blog-slice/index.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import React from 'react';
+import styles from './style.css';
+import BlogEntry from '../../../components/blog-entry';
+import type { BlogPost } from '../../../components/blog-entry';
+
+const blogSlice = ({ blogPosts, title }: { blogPosts: Array<BlogPost>, title: string }) => {
```

Comment:
In JSX, lower-case tag names (eg. `<blogPost>`) are considered to be HTML tags. Maybe we should always uppercase these names (it doesn't really matter in this instance as the export is default but still think its better to be consistent).